sap/5.7/modules/ROOT/pages/sap-connector-config-snc-kerberos.adoc

```diff
@@ -0,0 +1,149 @@
+
+. Configure the `krb5.conf` file using the KDC address and the realm information (SSODOMAIN.LAB). The `krb5.conf` file is created by default in the `etc` directory. 
++
+----
+[libdefaults]
+    # "dns_canonicalize_hostname" and "rdns" are better set to false for improved security.
+    # If set to true, the canonicalization mechanism performed by Kerberos client may
+    # allow service impersonification, the consequence is similar to conducting TLS certificate
+    # verification without checking host name.
+    # If left unspecified, the two parameters will have default value true, which is less secure.
+    dns_canonicalize_hostname = false
+    rdns = false
+    #   default_realm = EXAMPLE.COM
+        default_realm = DOMAIN.LOCAL
+        
+[realms]
+    #       EXAMPLE.COM = {
+    #                kdc = kerberos.example.com
+    #               admin_server = kerberos.example.com
+    #       }
+DOMAIN.LOCAL = {
+                kdc = DC.DOMAIN.LOCAL
+               admin_server = dc.DOMAIN.local
+       }
+[domain_realm]
+  .domain.local = DOMAIN.LOCAL
+   domain.local = DOMAIN.LOCAL
+
+[logging]
+    kdc = FILE:/var/log/krb5/krb5kdc.log
+    admin_server = FILE:/var/log/krb5/kadmind.log
+    default = SYSLOG:NOTICE:DAEMON
+
+[libdefaults]
+    default_ccache_name = FILE:/tmp/krb5cc_%{uid}
+----
++
+. Import the key to the `etc` directory.
++
+`ktutil`
++
+`wkt /etc/krb5.keytab`
++
+`q`
++
+. Obtain a Kerberos Ticket Granting Ticket (TGT).
++
+`kinit -V SAPService/sapdev@SSODOMAIN.LAB`
++
+. Change the permissions for the keytab.
++
+`chgrp sapsys /etc/krb5.keytab`
++
+`chmod 640 /etc/krb5.keytab`
++
+. Set up automatic renewal for the Kerberos TGT.
++
+`crontab –e 01 0,6,12,18 * * * /usr/bin/kinit -R SAPService/sapdev@SSODOMAIN.LAB`
++
+. Go to the location of the GSS-API library in the SAP server (in this example, it is under the `/usr/lib64/libgssapi_krb5.so.2` path) and configure the parameters for SNC with Kerberos in the Default Profile and then restart SAP.
+* `snc/permit_insecure_start = 1`
+* `snc/data_protection/use = 3`
+* `snc/data_protection/max = 3`
+* `snc/data_protection/min = 1`
+* `snc/accept_insecure_r3int_rfc = 1`
+* `snc/accept_insecure_rfc = 1`
+* `snc/accept_insecure_cpic = 1`
+* `snc/accept_insecure_gui = 1`
+* `snc/gssapi_lib = /usr/lib64/libgssapi_krb5.so.2`
+* `snc/enable = 1`
+* `snc/identity/as = p:SAPService/sapdev@SSODOMAIN.LAB`
+* `login/password_change_for_SSO = 0`
+. Create `adminsap` as a user in SAP and set the SNC name using the transaction `SU01`.
+. For the RFC destination, use the transaction `SM59`. In the tab logon and security, enable the SNC option and add the SPN name as a partner.
+
+== Configure SAP Connector
+
+Configure SAP Connector to enable SNC with Kerberos:
+
+. Access Anypoint Studio.
+. Select *SAP* in the Studio canvas.
+. Click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
```

Comment:
Consider using the icon. 
I recently updated some docs like this:
Click the Add icon (<icon>) _(alt: Add icon in Studio)_ for the Connector configuration field.